cpp/src/io/statistics/typed_statistics_chunk.cuh

```diff
@@ -244,9 +244,9 @@ get_untyped_chunk(typed_statistics_chunk<T, include_aggregate> const& chunk)
   stat.null_count = chunk.null_count;
   stat.has_minmax = chunk.has_minmax;
   stat.has_sum    = [&]() {
-    if (!chunk.has_minmax) return false;
     // invalidate the sum if overflow or underflow is possible
     if constexpr (std::is_floating_point_v<E> or std::is_integral_v<E>) {
+      if (!chunk.has_minmax) { return true; }
```

Comment:
is this change right?
For the same condition, return type is inverted.

Reply:
This part is meant to disable the sum when there's a chance of overflow. We had the old condtion because we treated the statistics parts as indivisible - either return both the sum and min/max, or none. This does not really need to be the case. So, now, we return the sum even without min/max. The new logic is - if there's no minmax, we have no valid elements and the sum can be returned (as zero).